README.md

```diff
@@ -0,0 +1,77 @@
+# Covid-19 data traker
```

Comment:
- [ ] Add screenshots for your app to this file. This will give users what to expect.

Reply:
DONE ✅